src/FluentValidation/Internal/PropertyRule.cs

```diff
@@ -30,6 +30,8 @@ namespace FluentValidation.Internal;
 /// </summary>
 internal class PropertyRule<T, TProperty> : RuleBase<T, TProperty, TProperty>, IValidationRuleInternal<T, TProperty> {
 
+	private Func<ValidationContext<T>, string> _displayNameFunc;

```

Comment:
This should be moved to `RuleBase` so `CollectionPropertyRule` can also make use of it

Reply:
Resolved in ccc3ebe81060a1f1c9f83fa1a232e42a38060ab3